script/upload-node-checksums.py

```diff
@@ -14,17 +16,23 @@
 
 def main():
   args = parse_args()
+  dist_url = args.dist_url
+  if dist_url[len(dist_url) - 1] != "/":
```

Comment:
`dist_url[-1]` should probably do the trick.

Reply:
Python 🎉